### PR TITLE
DRAFT: Sonar fixes for develop branch

### PR DESCRIPTION
**Pull request by mstead**
**NOTE: DO NOT MERGE**

Testing suppression.

### CODE REVIEW

File: swatch-core/src/main/java/org/candlepin/subscriptions/exception/mapper/BadRequestExceptionMapper.java [hunk 1]
```diff
@@ -20,11 +20,11 @@
  */
 package org.candlepin.subscriptions.exception.mapper;
 
+import javax.ws.rs.BadRequestException;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.ext.Provider;
 import org.candlepin.subscriptions.exception.ErrorCode;
 import org.candlepin.subscriptions.utilization.api.model.Error;
-import org.jboss.resteasy.spi.BadRequestException;
```
Comment (lindseyburnett) [inline swatch-core/src/main/java/org/candlepin/subscriptions/exception/mapper/BadRequestExceptionMapper.java:27]: Was this change on purpose for sonar?

**Comment by mstead** [comment]
Waiting to see what sonar does with these changes applied. I want to make sure that the commented out one still fails because I don't see it locally in the IDE.
